scipy/ndimage/_morphology.py

```diff
@@ -1922,12 +1922,93 @@ def distance_transform_bf(input, metric="euclidean", sampling=None,
         Returned only when `return_indices` is True and `indices` is not
         supplied.
 
+    See Also
+    --------
+    distance_transform_cdt : Faster distance transform for taxicab and
+        chessboard metrics
+    distance_transform_edt : Faster distance transform for euclidean metric
+
     Notes
     -----
     This function employs a slow brute force algorithm, see also the
     function distance_transform_cdt for more efficient taxicab and
     chessboard algorithms.
 
+    Examples
+    --------
+    Import the necessary modules.
+
+    >>> import numpy as np
+    >>> from scipy.ndimage import distance_transform_bf
+    >>> import matplotlib.pyplot as plt
+    >>> from mpl_toolkits.axes_grid1 import ImageGrid
+
+    Compare the distance transforms using different metrics on the same
+    image. First, we create a toy binary image.
+
+    >>> def add_circle(center_x, center_y, radius, image, fillvalue=1):
+    ...     # fill circular area with 1
+    ...     xx, yy = np.mgrid[:image.shape[0], :image.shape[1]]
+    ...     circle = (xx - center_x) ** 2 + (yy - center_y) ** 2
+    ...     circle_shape = np.sqrt(circle) < radius
+    ...     image[circle_shape] = fillvalue
+    ...     return image
+
+    >>> image = np.zeros((100, 100), dtype=np.uint8)
+    >>> image[35:65, 20:80] = 1
+    >>> image = add_circle(28, 65, 10, image)
+    >>> image = add_circle(37, 30, 10, image)
+    >>> image = add_circle(70, 45, 20, image)
+    >>> image = add_circle(45, 80, 10, image)
+    >>> fig, ax = plt.subplots()
+    >>> ax.imshow(image, cmap='gray')
+    >>> ax.axis('off')
+    >>> ax.set_title("Binary test image: foreground in white")
```

Comment:
There is no plt.show() here so it shows up with the other figure. That said I am no sure it is necessary to show it twice?

![image](https://user-images.githubusercontent.com/60778417/231595963-b80c5330-1d86-4c0b-af1c-b21ecb4ebbb7.png)

Reply:
Agreed 👍 The image grid is enough.